odetoolbox/__init__.py

```diff
@@ -110,12 +95,13 @@ def _from_json_to_shapes(indict, options_dict, parameters=None) -> List[Shape]:
     all_parameter_symbols = set()
     all_variable_symbols_ = set()
     for shape_json in indict["dynamics"]:
-        shape = Shape.from_json(shape_json, time_symbol=options_dict["input_time_symbol"], differential_order_symbol=options_dict["differential_order_symbol"], parameters=parameters)
+        shape = Shape.from_json(shape_json, time_symbol=Config().input_time_symbol, parameters=parameters)
         all_variable_symbols.extend(shape.get_state_variables())
-        all_variable_symbols_.update(shape.get_state_variables(derivative_symbol="__d"))
+        all_variable_symbols_.update(shape.get_state_variables(derivative_symbol=Config().differential_order_symbol))
```

Comment:
Do we need to pass the `Config` options here as it is a global class that can be accessed directly in the function as well? This applies to all the places where `Config` is passed as a function argument.

Reply:
Thanks for noticing this, I cleaned it up in another place, but in the remainder of cases ``Shape.get_state_variables()`` is called with different values of this parameter.